tests/test_auth.py

```
@@ -222,3 +228,41 @@ def test_get_github_headers_no_token(self):
+        monkeypatch.setattr(AuthService, "obs_credentials_file", temp_obs_file)
+        
+        with patch.dict(os.environ, {
+            'OBS_HOST': '127.0.0.1',
+            'OBS_PORT': '4455',
+            'OBS_PASSWORD': 'test_password',
+            'OBS_SCENE': 'test_scene',
+            'OBS_DRY_RUN': 'false'
+        }):
+            auth_service = AuthService()
+            credentials = auth_service.get_obs_credentials()
+            assert credentials is not None
```

Ruff (0.12.2)

Use of `assert` detected

(S101)